corehq/apps/userreports/tasks.py

```diff
@@ -85,6 +85,16 @@ def rebuild_indicators(indicator_config_id, initiated_by=None, limit=-1, source=
         send = toggles.SEND_UCR_REBUILD_INFO.enabled(initiated_by)
     with notify_someone(initiated_by, success_message=success, error_message=failure, send=send):
         adapter = get_indicator_adapter(config)
+
+        if engine_id:
+            if getattr(adapter, 'all_adapters', None):
```

Comment:
Why not do a type check of `MultiDBSqlAdapter`?

Reply:
one less import? also just following the pattern you used.

Reply:
👍